Schronisko_app.py

```diff
@@ -0,0 +1,129 @@
+from flask import Flask, request, render_template, g, flash, redirect, url_for, abort
+
+import sqlite3
+
+DATABASE = '/path/to/database.db'
+app = Flask(__name__)
+app.secret_key = "TjWnZq4t7w!z%C*F"
+app_info = {
+    "db_file": "database/schronisko.db"
+}
```

Comment:
Please put secrets in `.env`, and its template in `env_template`. You can load db fpath from env variables too